src/core/interfaces/include-object.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { WhereType } from "../types";
+
+/**
+ * Interface for generated include objects.
+ */
+export interface IncludeObject {
```

Comment:
Just a suggestion. Take a look at IncludeOptions from sequelize models. This is the native sequelize interface for includes. You can use IncludeOptions instead of creating an interface with same params. Or you can create a type from it